tests/unittests/modules/test_time_sync.py

```diff
@@ -0,0 +1,139 @@
+        """
+        block_dict={"args": {"ntp_servers": [
+            "server1", "server2"
+        ]}}
+        check_id = "test-1"
+
+        res = time_sync.get_filtered_params_to_log(check_id, block_dict)
+        self.assertEqual(res, {"ntp_servers": [
+            "server1", "server2"]})
+
+    def test_execute1(self):
+        """
+        Sample execution. Having 5 ntp servers
+        """
+        block_dict={
+            'args': {
+                "ntp_servers": [
+                    "server1", "server2", "server3", "server4", "server5"
+                ]
+            }
+        }
+        check_id = "test-1"
+        test_obj = TestClass()
+        test_obj.offset = 5
+
+        with patch('hubblestack.extmods.hubble_mods.time_sync.ntplib.NTPClient') as ntplib_mock:
+            ntplib_mock.return_value.request.return_value = test_obj
+
+            status, res = time_sync.execute(check_id, block_dict)
+            self.assertEqual(status, True)
+
+    def test_execute2(self):
+        """
+        Sample execution. Having 2 ntp servers
+        """
+        block_dict={
+            'args': {
+                "ntp_servers": [
+                    "server1", "server2"
+                ],
+                "nb_servers": 2
+            }
+        }
+        check_id = "test-1"
+        test_obj = TestClass()
+        test_obj.offset = 5
+
+        with patch('hubblestack.extmods.hubble_mods.time_sync.ntplib.NTPClient') as ntplib_mock:
+            ntplib_mock.return_value.request.return_value = test_obj
+
+            status, res = time_sync.execute(check_id, block_dict)
+            self.assertEqual(status, True)
+
+    def test_execute3(self):
+        """
+        Sample execution having less number of ntp servers.
+        Should fail
+        """
+        block_dict={
+            'args': {
+                "ntp_servers": [
+                    "server1", "server2"
+                ]
+            }
+        }
+        check_id = "test-1"
+        test_obj = TestClass()
+        test_obj.offset = 5
+
+        with patch('hubblestack.extmods.hubble_mods.time_sync.ntplib.NTPClient') as ntplib_mock:
+            ntplib_mock.return_value.request.return_value = test_obj
+
+            status, res = time_sync.execute(check_id, block_dict)
+            self.assertEqual(status, False)
+
+    def test_execute4(self):
+        """
+        offset is more than allowed
+        Should fail
+        """
+        block_dict={
+            'args': {
+                "ntp_servers": [
+                    "server1", "server2"
+                ]
+            }
+        }
+        check_id = "test-1"
+        test_obj = TestClass()
+        test_obj.offset = 1000
+
+        with patch('hubblestack.extmods.hubble_mods.time_sync.ntplib.NTPClient') as ntplib_mock:
+            ntplib_mock.return_value.request.return_value = test_obj
+
+            status, res = time_sync.execute(check_id, block_dict)
+            self.assertEqual(status, True)
```

Comment:
it should be false in case of failure

Reply:
First is the module status. Its true as it ran successfully. The value(return value) is False.
This is as per old module